src/components/PriceFilter.jsx

```diff
@@ -62,7 +62,7 @@ function PriceFilter({ priceFilter = { min: 0, max: 1000 }, updatePriceFilter, c
                         {" "}
                         Above $250{" "}
                     </button>
-                    <button className="bg-gray-300 hover:bg-gray-400 text-gray-800 font-bold py-2 px-4 rounded" onClick={clearPriceFilter}>Clear Filter</button>
+                    <button className="border rounded-md bg-white border-primary-600 text-primary-300 font-bold py-2 px-4 rounded hover:bg-primary-300 hover:text-white py-2 px-4" onClick={clearPriceFilter}>Clear Filter</button>
```

Comment:
text bold is not in line with any other style - should that be removed?

Reply:
1. I added it in to differentiate, but Anastasia is going to come in and fix styling for these. 
2. I am going to leave it as is for now so we can get on with other things. Given it doesn't do anything when no filters are active, it's not as bad as the apply filter one where it was breaking the gifts rendering

Reply:
Updated typo in header.

Reply:
1. if Anastasiia is going to address button style - I accept it.
2. not a nice user experience but true that functionality wise is not breaking like the other one 
3. thanks for fixing that typo